oddt/scoring/descriptors/__init__.py

```diff
@@ -207,6 +209,68 @@ def __reduce__(self):
                                            self.aligned_pairs)
 
 
+class universal_descriptor(object):
+    def __init__(self,
+                 func,
+                 protein=None,
+                 shape=None):
+        """Close contacts descriptor which tallies atoms of type X in certain
+        cutoff from atoms of type Y.
+
+        Parameters
+        ----------
+            func: object
+                A function to be mapped accross all ligands. Can be any callable
```

Comment:
Any callable object? I guess it should at least take some well-defined arguments (`ligand` and optionally `protein`?) and return something that can be v-stacked

Reply:
Yeah, I could be more precise.